### DISCUSSION TITLE
low cpu frequency

### DISCUSSION THREAD

**Discussion by constantius24** [itsManjeet/avyos#96] [General]
hi. I have installed rlxos on core i5-4590 quad-core cpu. But cpu frequency all the time is low 800MHZ. Something wrong is with cpu governor. System is working slow due to low cpu frequency.

**Comment by ghost** [comment]
rlxos use CPU P state for frequency, you can use `powerprofilectl` to manipulate the power profile,

By default Balanced, or powersaver depending on system specifications
